docs/2.drivers/null.md

```diff
@@ -0,0 +1,20 @@
+---
+icon: bi:trash3-fill
+---
+
+# Null
+
+> Discards all data.
+
+::warning
+This driver does NOT store any data. It will discard any data written to it and will always return null!
```

Comment:
```suggestion
This driver does NOT store any data. It will discard any data written to it and will always return null similar to [`/dev/null`](https://en.wikipedia.org/wiki/Null_device)
```